src/c_api/c_predict_api.cc

```diff
@@ -17,6 +17,16 @@
 
 using namespace mxnet;
 
+namespace mxnet {
+namespace op {
+void RegisterLegacyOpProp();
+}
+const std::vector<std::string> kHiddenKeys = {
+  "ctx_group", "lr_mult", "wd_mult", "__force_mirroring__"
```

Comment:
what's this for? This part of code was updated. You need to copy again.
Or better, fix it so no copying is needed